src/inverseRobot.js

```diff
@@ -1,13 +1,33 @@
 'use strict';
 
-/*
- * @param {object} robot
- *
- * @return {object}
-*/
+const robert = {
+  Robert: 'name',
+  123: 'chipVer',
+  113: 'chipVerPro',
+};
+
+inverseRobot(robert);
 
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  if (robot.hasOwnProperty('name') && robot.name !== 'name') {
+    inversedRobot[robot.name] = 'name';
+  }
```

Comment:
Why do you need to check the name separately? 
Can't you check it with all other properties?